cupy/cuda/memory.pyx

```diff
@@ -676,9 +676,12 @@ cdef class PooledMemory(BaseMemory):
 cdef size_t _index_compaction_threshold = 512
```

Comment:
@emcastillo Could you check with the team if this line is coupled to the alignment? No comments for how it was chosen...